### ISSUE TITLE
CSV::MalformedCSVError when reading from file

### ISSUE THREAD

**Issue by clebrun** [clebrun/Distributator#26]
Read gives a `CSV::MalformedCSVError`. Need to change the converter to leave quoted fields, and might as well change the other converter methods to quote fields for consisitency. Discovered in #19

**Comment by clebrun** [comment]
in @6ed5c6bfebf5a1c70d326cfc072d4abaaf64bf14 I changed the .rename_header lambda to output quoted headers. This doesn't fix the error: “unquoted fields do not allow \r or \n”, but it's the only change that happens before #read is called.

 After some digging, I came up with a few SO threads about Excel [formatting saved .csv files weirdly](https://stackoverflow.com/questions/9200175/rails-fastercsv-unquoted-fields-do-not-allow-r-or-n). I thought I had addressed this with #25, but it looks like there are still issues here. There is a solution I'm working on that intercepts files, reads them and encodes them before passing them on, but it needs work.
